src/HotChocolate/Core/src/Types.Scalars/LatitudeType.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using HotChocolate.Language;
+
+namespace HotChocolate.Types
+{
+    /// <summary>
+    /// The `LatitudeType` scalar represents a valid decimal degrees latitude number.
+    /// <a>https://en.wikipedia.org/wiki/Latitude</a>
+    /// </summary>
+    public class LatitudeType : ScalarType<double, StringValueNode>
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="LatitudeType"/>
+        /// </summary>
+        public LatitudeType()
+            : this(
+                WellKnownScalarTypes.Latitude,
+                ScalarResources.LatitudeType_Description)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="LatitudeType"/>
+        /// </summary>
+        public LatitudeType(
+            NameString name,
+            string? description = null,
+            BindingBehavior bind = BindingBehavior.Explicit)
+            : base(name, bind)
+        {
+            Description = description;
+        }
+
+        /// <inheritdoc />
+        public override IValueNode ParseResult(object? resultValue)
+        {
+            return resultValue switch
+            {
+                null => NullValueNode.Default,
+
+                string s when Latitude.IsSexagesimal(s) &&
```

Comment:
I think we do not need the `Latitude.IsSexagesimal(s)` as we already to the parsing in `TryDeserializeFromString`